render/wlr_renderer.c

```diff
@@ -189,6 +189,8 @@ void wlr_renderer_init_wl_display(struct wlr_renderer *r,
 	if (r->impl->init_wl_display) {
 		r->impl->init_wl_display(r, wl_display);
```

Comment:
We should probably also make `impl->init_wl_display` return a `bool`.

Reply:
Thanks, done in the new commit.